### DISCUSSION TITLE
Request for comment: Conversations UI Kit

### DISCUSSION THREAD

**Discussion by shleewhite** [twilio-labs/paste#2477] [Show and tell]
We're excited to be releasing the first versions of the components for the Conversations UI Kit! Since these components are entirely new and are still works in progress, we would appreciate all discussions around them to be in one single discussion topic.

![Screen Shot 2022-06-21 at 3 18 26 PM](https://user-images.githubusercontent.com/8482334/174906439-f8e5e6d6-44cd-4f56-b239-655950de1f51.png)

## What is the Conversations UI Kit?

It will be a group of components for [the Twilio Conversations API ](https://www.twilio.com/conversations-api) to create accessible and scalable chat experiences. These components are in `@twilio-paste/core/chat-log` and include:
* ChatMessage
* ChatBubble
* ChatMessageMeta

The designs for all the phase 1 components can be found [in Figma](https://www.figma.com/file/cSdKlqoTbxoAUmnrYkfFWl/Conversations-UI-Kit?node-id=0%3A1).

If you have any questions or feedback about these components, please post them as comments below.

**Comment by RvPr** [comment]
Hi @shleewhite . I see that there's also a different twilio-conversations demo application that comes with a UI kit as well:

- https://www.twilio.com/code-exchange/conversations-quick-deploy
- https://github.com/twilio/twilio-conversations-demo-react

> This application demonstrates a basic Conversations API client application with the ability to create and join conversations, add other participants into the conversations and exchange messages. **It also includes a UI Kit that can be customized for your production application.**

Do you have recommendations on which UI kit would be better for us to use as a starting point for our app?

**Reply by shleewhite** [reply]
Hi @RvPr! 

The Paste Conversations UI Kit is meant to be complimentary to these other demo applications. The Paste components are presentational, meaning they are meant to help with the layout and styling of a chat UI. Eventually, these demo apps will use the Paste Chat Log components, but they have not been updated yet.

If you want, you can use the Paste Chat Log components in either demo app because they already have Paste as a dependency.  

Hopefully this helps clarify what the Paste components are for!
